source/includes/fact-download-queries.rst

```diff
@@ -0,0 +1,54 @@
+Allows you to download completed conversions in the 
+:guilabel:`Query Converter` pane.
+
+Individual Query
+~~~~~~~~~~~~~~~~
+
+.. procedure::
+   :style: normal
+
+   .. step:: Select your query
+
+      a. From the :guilabel:`Code Generation` tab, click the 
+         :guilabel:`Query Converter` pane.
+
+         .. note::
+
+           If you are not already logged into your Atlas account or have
+           an expired session, you must login to proceed.
```

Comment:
(terminology)
s/b "log in" 
Always confusing. 
login = noun
log in = verb

Reply:
Good catch! I'll fix here and do a sweep of the usage in the doc as a lower-priority fast-follow.